workflows/ampliseq.nf

```diff
@@ -365,6 +368,16 @@ workflow AMPLISEQ {
     }
 
     //
+    // Modules : Filtering based on codons in an open reading frame
+    //
+    if (params.filter_codons || params.orf_start || params.orf_end || params.stop_codons) {
```

Comment:
If you change the default of those params then that would need changing as well.

Reply:
Yes! I think so. I guess I will keep it so that one has to use `--filter_codons`! That makes it explicit and I will change the defaults.